server/utils/assessments/utils.ts

```diff
@@ -287,6 +287,24 @@ const getSectionSuffix = (task: Task) => {
 const decisionFromAssessment = (assessment: Assessment) =>
   assessment?.data?.['make-a-decision']?.['make-a-decision']?.decision || ''
 
+const confirmationPageMessage = (assessment: Assessment) => {
+  switch (decisionFromAssessment(assessment)) {
+    case 'releaseDate':
+      return `<p>We've notified the Probation Practitioner that this application has been assessed as suitable.</p>
+      <p>The assessment can now be used to match Robert Brown to a bed in an Approved Premises.</p>`
+      break
+    case 'hold':
+      return `<p>We've notified the Probation Practitioner that this application has been assessed as suitable.</p>
+      <p>This case is now paused until the oral hearing outcome has been provided by the Probation Practitioner and a release date is confirmed.</p>
+      <p>It will be added to the matching queue if the oral hearing is successful.</p>`
+      break
+    default:
+      return `<p>We've sent you a confirmation email.</p>
```

Comment:
We should probably confirm that sending confirmation emails is in scope for this round of work. I don't think it's a huge job (thanks GOV.UK Notify!), but we need to factor it in